packages/angular/src/generators/application/lib/add-e2e.ts

```diff
@@ -10,23 +10,27 @@ import {
 } from '@nx/devkit';
 import { nxVersion } from '../../../utils/versions';
 import type { NormalizedSchema } from './normalized-schema';
-import { cypressProjectGenerator } from '@nx/cypress';
+import { configurationGenerator } from '@nx/cypress';
 
 export async function addE2e(tree: Tree, options: NormalizedSchema) {
   if (options.e2eTestRunner === 'cypress') {
     // TODO: This can call `@nx/web:static-config` generator when ready
     addFileServerTarget(tree, options, 'serve-static');
-
-    await cypressProjectGenerator(tree, {
-      name: options.e2eProjectName,
+    addProjectConfiguration(tree, options.e2eProjectName, {
+      projectType: 'application',
+      root: options.e2eProjectRoot,
+      sourceRoot: joinPathFragments(options.e2eProjectRoot, 'src'),
+      targets: {},
+      tags: [],
+      implicitDependencies: [options.name],
+    });
+    await configurationGenerator(tree, {
+      project: options.e2eProjectName,
       directory: options.e2eProjectRoot,
```

Comment:
I think this is wrong. The `directory` option in the `configurationGenerator` is meant to be and quote "A directory where the project is placed relative from the project root". The same applies to the rest of generators updated in this PR providing the project root as the directory.
```suggestion
```